db/migrate/20151020033930_add_relation_between_user_and_neighborhood.rb

```diff
@@ -0,0 +1,9 @@
+class AddRelationBetweenUserAndNeighborhood < ActiveRecord::Migration
+  def change
```

Comment:
@HacksForHugs This isn't adding relations between user and neighborhood. It seems to be adding relations between group and neighborhood. Rename?